code_benches/run.py

```diff
@@ -105,17 +149,18 @@ def sources(self):
     # == Custom Benchmarks ==
     Program("custom_binarytrees", [16], 2 ** 14),
     Program("custom_function_pointers", [], 2 ** 14),
-    Program("custom_libjpeg", [], 2 ** 15,
-            custom_arguments=["-Wno-incompatible-library-redeclaration", "-Wno-implicit-function-declaration",
-                              "-Wno-shift-negative-value"]),
+    #Program("custom_libjpeg", [], 2 ** 15,
+    #        custom_arguments=["-Wno-incompatible-library-redeclaration", "-Wno-implicit-function-declaration",
+    #                          "-Wno-shift-negative-value"]),
     Program("custom_matrix_multiply", [], 2 ** 14),
     Program("custom_memcmp", [], 2 ** 14),
-    Program("custom_sqlite", [], 2 ** 15, custom_arguments=["-DSQLITE_MUTEX_NOOP", "-ldl"]),
+    # TODO need to remove dependency on posix headers
+    #Program("custom_sqlite", [], 2 ** 15, custom_arguments=["-DSQLITE_MUTEX_NOOP", "-ldl"]),
```

Comment:
Broken with wasi?

Reply:
Ah yes, this benchmark was using a number of Linux-specific headers (maybe based on OS checks), that wasi-sdk rejects while wasmception lets compile.

I haven't looked that much into it, but I think it's related to wasmception being based on syscalls whereas wasi-sdk is based on Wasm imports. wasmception's syscalls would turn into a runtime failure if called, whereas wasi-sdk's imports turn into a link-time failure in aWsm.

Do you all have experience configuring sqlite?

Reply:
Fix not needed for this PR